.github/scripts/hermetic_library_generation.sh

```diff
@@ -106,9 +106,11 @@ if [[ "${showcase_mode}" == "true" ]]; then
 fi
 
 # get changed library list.
-changed_libraries=$(python hermetic_build/common/cli/get_changed_libraries.py create \
+changed_libraries_file=$(mktemp)
+python hermetic_build/common/cli/get_changed_libraries.py create \
   --baseline-generation-config-path="${baseline_generation_config}" \
-  --current-generation-config-path="${generation_config}")
+  --current-generation-config-path="${generation_config}" | tee > "${changed_libraries_file}"
+changed_libraries=$(cat "${changed_libraries_file}")
 echo "Changed libraries are: ${changed_libraries:-"No changed library"}."
```

Comment:
I suspect this double quotations included in double quotations. Check this before submitting this.

Reply:
I surrounded the whole variable with double quotes now.